src/subcommand/viz_main.cpp

```diff
@@ -534,10 +542,7 @@ namespace odgi {
                 uint64_t min_x = std::numeric_limits<uint64_t>::max();
                 uint64_t max_x = std::numeric_limits<uint64_t>::min(); // 0
 
-                // In binned mode, the min/max_x values changes based on the bin width
-                if (!args::get(binned_mode)) {
-                    _bin_width = 1;
-                }
+                // In binned mode, the min/max_x values changes based on the bin width; in standard mode, _bin_width is 1, so nothing change here
```

Comment:
I think this is clear from your documentation above. We can delete this.

Reply:
The comment is related to the `_bin_width` variable that I added to the denominators of the following lines of code, but if you click on the last commit on git, those lines are not visible and the comment seems unnecessary.

Reply:
I somehow missed that, sorry! So all good.